web/src/js/ducks/flows.js

```diff
@@ -20,32 +23,54 @@ const defaultState = {
     view: [],
     filter: undefined,
     highlight: undefined,
+    sort: {sortColumn: undefined, sortDesc: false},
 }
 
 function makeFilterFn(filter) {
     return filter ? Filt.parse(filter) : () => true;
 }
 
+
+function makeSortFn(sort){
+    let column = _.find(flowtable_columns, c => c.name == sort.sortColumn);
```

Comment:
i cannot find a dict in ../components/flowtable-columns should i create one ? i found an array 
`var all_columns = [
    TLSColumn,
    IconColumn,
    PathColumn,
    MethodColumn,
    StatusColumn,
    SizeColumn,
    TimeColumn
]`

and if a just use `    let c1 = columns[sort.sortColumn];` i got undefined.